src/components/Incorporation/SummaryDefineCompany.vue

```diff
@@ -23,47 +23,50 @@
           <v-col cols="12" sm="3" class="pr-4">
             <label>Name</label>
           </v-col>
-          <v-col cols="12" sm="9" class="pr-4">
+          <v-col cols="12" sm="9">
```

Comment:
This is the last column in this row. It's doesn't need a right gutter; it can run right to the edge of this component.